src/cordova/apple/OutlineAppleLib/Package.swift

```diff
@@ -69,8 +69,7 @@ let package = Package(
         ),
         .binaryTarget(
             name: "Tun2socks",
-            url: "https://github.com/Jigsaw-Code/outline-go-tun2socks/releases/download/v3.4.0/apple.zip",
-            checksum: "6c6880fa7d419a5fddc10588edffa0b23b5a44f0f840cf6865372127285bcc42"
+            path: "../../../../output/build/apple/tun2socks.xcframework"
```

Comment:
This needs some discussion. We should have the component in the path, so it's easier to delete the component's output. Otherwise it's hard to tell what files come from what component. So the path would be `output/build/apple/tun2socks/tun2socks.xcframework`

But it's also ok if we drop the platform, and put all the platform-specific output in the same component dir:
`output/build/tun2socks/tun2socks.xcframework`